app/experimenter/experiments/tests/test_models/test_models_nimbus.py

```diff
@@ -24,21 +24,75 @@ def test_str(self):
         experiment = NimbusExperimentFactory.create(slug="experiment-slug")
         self.assertEqual(str(experiment), experiment.name)
 
-    def test_targeting_config_not_set(self):
+    def test_targeting_for_experiment_without_channels(self):
         experiment = NimbusExperimentFactory.create(
-            targeting_config_slug=NimbusExperiment.TargetingConfig.NO_TARGETING
+            firefox_min_version=NimbusExperiment.Version.FIREFOX_83,
+            targeting_config_slug=NimbusExperiment.TargetingConfig.ALL_ENGLISH,
+            application=NimbusExperiment.Application.DESKTOP,
+            channel=NimbusExperiment.Channel.NO_CHANNEL,
         )
-        self.assertIsNone(experiment.targeting_config)
 
-    def test_targeting_config_set(self):
+        self.assertEqual(
+            experiment.targeting,
+            (
+                "version|versionCompare('83.!') >= 0 "
+                "&& localeLanguageCode == 'en' "
+                "&& 'app.shield.optoutstudies.enabled'|preferenceValue"
+            ),
+        )
+
+    def test_serializer_outputs_expected_targeting_for_mobile(self):
```

Comment:
`serializer` doesn't apply here, maybe replace that with the method name?

Reply:
ah whoops